packages/iTwinUI-react/src/core/Slider/Slider.tsx

```diff
@@ -405,9 +433,11 @@ export const Slider = React.forwardRef<HTMLDivElement, SliderProps>(
         return (
           <div className='iui-slider-ticks'>
             {tickLabels.map((label, index) => (
-              <span key={index} className='iui-slider-tick'>
-                {label}
-              </span>
+              // TODO: Is passing index alright to pass as a key? I guess it is since there's nothing better, but just confirming.
```

Comment:
Yeah, it is not recommended to use index as a key but sometimes there is no better property to use. Here you could combine index with label ``{`${index-label}`}`` if label was a string but it is `React.ReactNode`.

Reply:
**[DONE]** Thanks